integration-mc1-15/src/main/kotlin/org/eln2/utils/OreData.kt

```diff
@@ -0,0 +1,13 @@
+package org.eln2.utils
+
+import net.minecraft.item.ItemStack
+
+/**
+ * OreData
+ * Data about the ores you are trying to make.
+ *
+ * @param hardness How hard is the block (pickaxe type basically)
+ * @param name Name of the ore (eg, "native_copper")
+ * @param result The item to drop
+ */
+data class OreData(val hardness: Float, val name: String, val result: ItemStack)
```

Comment:
Can we use an enum instead (for hardness), just to reduce confusion? Hand, wood, stone, iron, diamond...

Reply:
Does Minecraft already have such an enum? I am unfamiliar with how hardness works, I thought it affected blast resistance mostly and also the level at which an iron pick is required.

Reply:
It probably has constants, but those don't make it through the decompiler.

Reply:
Time to make an enum then. Any documentation you know of that documents what those hardness levels are?

Reply:
Well, there's https://minecraft.gamepedia.com/Module:Hardness_values

Reply:
Actually, nevermind. Apparently, the block hardness affects both the tool and the mining speed, and many of the blocks in Minecraft have values all over the darned place. We could add constants that have particular minimum values for say an iron pickaxe, but there is no strict set of values they seem to be using.

https://minecraft.gamepedia.com/Breaking#Blocks_by_hardness